.circleci/config.yml

```
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  elixir: membraneframework/elixir@1
+  elixir: membraneframework/elixir@1.3.0
```

This will prevent automatical usage of the newer minor versions, please keep it as it was (it will use the latest 1.3.0)

```suggestion
  elixir: membraneframework/elixir@1
```